.github/workflows/mvn-verify-check.yml

```diff
@@ -52,42 +63,34 @@ jobs:
         with:
           path: ~/.m2
           key: ${{ env.dailyCacheKey }}
+          restore-keys: ${{ runner.os }}-maven-
       - name: populate-daily-cache
         if: steps.cache.outputs.cache-hit != 'true'
         run: |
           set -x
           max_retry=3; delay=30; i=1
           while true; do
-            for pom in pom.xml scala2.13/pom.xml; do 
-              mvn -f $pom help:evaluate -pl dist -Dexpression=included_buildvers \
-              -DforceStdout -PnoSnapshots -q | tr -d ',' | \
-              xargs -n 1 bash -c 'mvn -f $1 initialize -pl sql-plugin-api -am -Dbuildver=$2' _ $pom;
+            for pom in pom.xml scala2.13/pom.xml
+            do
+              # compile base versions to cache scala compiler and compiler bridge
+              mvn ${{ env.COMMON_MVN_FLAGS }} --file $pom \
+                process-test-resources -pl sql-plugin-api -am
```

Comment:
Should this be part of the for loop just after this (with the same loop setup) so it can benefit from retry logic?  It seems to defeat some of the dependency resolving since this will start to download dependencies, at least of the sql-plugin-api project.

Reply:
done @jlowe